Patch that applies modifications after running 'autoreconf -vif'
==========================================================================
--- a/dummy_drv_video/Makefile.in
+++ b/dummy_drv_video/Makefile.in
@@ -91,6 +91,7 @@
   sed '$$!N;$$!N;$$!N;$$!N;s/\n/ /g'
 am__installdirs = "$(DESTDIR)$(dummy_drv_video_ladir)"
 LTLIBRARIES = $(dummy_drv_video_la_LTLIBRARIES)
+dummy_drv_video_la_LIBADD =
 am_dummy_drv_video_la_OBJECTS = dummy_drv_video.lo object_heap.lo
 dummy_drv_video_la_OBJECTS = $(am_dummy_drv_video_la_OBJECTS)
 dummy_drv_video_la_LINK = $(LIBTOOL) --tag=CC $(AM_LIBTOOLFLAGS) \
@@ -258,8 +259,6 @@
 dummy_drv_video_la_LTLIBRARIES = dummy_drv_video.la
 dummy_drv_video_ladir = @LIBVA_DRIVERS_PATH@
 dummy_drv_video_la_LDFLAGS = -module -avoid-version -no-undefined -Wl,--no-undefined
-dummy_drv_video_la_LIBADD = $(top_builddir)/va/$(libvabackendlib)
-dummy_drv_video_la_DEPENDENCIES = $(top_builddir)/va/$(libvabackendlib)
 AM_CFLAGS = -I$(top_srcdir)/va -I$(top_srcdir)/../../include/external/ -I$(top_srcdir)/../../include/kmd
 dummy_drv_video_la_SOURCES = dummy_drv_video.c object_heap.c
 noinst_HEADERS = dummy_drv_video.h object_heap.h
--- a/test/basic/Makefile.in
+++ b/test/basic/Makefile.in
@@ -270,7 +270,9 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 AM_CFLAGS = -I$(top_srcdir)/va -I$(top_srcdir)/src/x11
-TEST_LIBS = $(top_builddir)/va/$(libvabackendlib)
+TEST_LIBS = $(top_builddir)/va/$(libvabackendlib) \
+  $(top_builddir)/va/$(libvacorelib)
+
 test_01_LDADD = $(TEST_LIBS)
 test_01_SOURCES = test_01.c
 test_02_LDADD = $(TEST_LIBS)
--- a/test/decode/Makefile.in
+++ b/test/decode/Makefile.in
@@ -231,7 +231,9 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 AM_CFLAGS = -I$(top_srcdir)/va -I$(top_srcdir)/src/x11
-TEST_LIBS = $(top_builddir)/va/$(libvabackendlib)
+TEST_LIBS = $(top_builddir)/va/$(libvabackendlib) \
+  $(top_builddir)/va/$(libvacorelib)
+
 mpeg2vldemo_LDADD = $(TEST_LIBS)
 mpeg2vldemo_SOURCES = mpeg2vldemo.c
 all: all-am
--- a/test/encode/Makefile.in
+++ b/test/encode/Makefile.in
@@ -231,7 +231,9 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 AM_CFLAGS = -I$(top_srcdir)/va -I$(top_srcdir)/src/x11
-TEST_LIBS = $(top_builddir)/va/$(libvabackendlib)
+TEST_LIBS = $(top_builddir)/va/$(libvabackendlib) \
+  $(top_builddir)/va/$(libvacorelib)
+
 h264encode_LDADD = $(TEST_LIBS)
 h264encode_SOURCES = h264encode.c
 all: all-am
--- a/test/putsurface/Makefile.in
+++ b/test/putsurface/Makefile.in
@@ -71,7 +71,8 @@
 PROGRAMS = $(bin_PROGRAMS)
 am_putsurface_OBJECTS = putsurface.$(OBJEXT)
 putsurface_OBJECTS = $(am_putsurface_OBJECTS)
-am__DEPENDENCIES_1 = $(top_builddir)/va/$(libvabackendlib)
+am__DEPENDENCIES_1 = $(top_builddir)/va/$(libvabackendlib) \
+	$(top_builddir)/va/$(libvacorelib)
 putsurface_DEPENDENCIES = $(am__DEPENDENCIES_1)
 DEFAULT_INCLUDES = -I.@am__isrc@ -I$(top_builddir)
 depcomp = $(SHELL) $(top_srcdir)/depcomp
@@ -232,7 +233,10 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 AM_CFLAGS = -I$(top_srcdir)/va -I$(top_srcdir)/src/x11
-TEST_LIBS = $(top_builddir)/va/$(libvabackendlib) -lpthread
+TEST_LIBS = $(top_builddir)/va/$(libvabackendlib) \
+  $(top_builddir)/va/$(libvacorelib) \
+  -lpthread
+
 putsurface_LDADD = $(TEST_LIBS)
 putsurface_SOURCES = putsurface.c
 EXTRA_DIST = loadsurface.h
--- a/va/Makefile.in
+++ b/va/Makefile.in
@@ -339,9 +339,9 @@
 libva_x11_backend = libva-x11.la
 libva_x11_backenddir = x11
 libva_x11_la_SOURCES = 
-libva_x11_la_LIBADD = $(libvacorelib) x11/libva_x11.la $(LIBVA_LIBS) $(X11_LIBS) $(XEXT_LIBS) $(DRM_LIBS) $(XFIXES_LIBS) -ldl
+libva_x11_la_LIBADD = x11/libva_x11.la $(LIBVA_LIBS) $(X11_LIBS) $(XEXT_LIBS) $(DRM_LIBS) $(XFIXES_LIBS) -ldl
 libva_x11_la_LDFLAGS = $(LDADD)
-libva_x11_la_DEPENDENCIES = $(libvacorelib) x11/libva_x11.la
+libva_x11_la_DEPENDENCIES = x11/libva_x11.la
 @BUILD_DUMMY_BACKEND_FALSE@libva_dummy_backend = 
 @BUILD_DUMMY_BACKEND_TRUE@libva_dummy_backend = libva-dummy.la
 @BUILD_DUMMY_BACKEND_FALSE@libva_dummy_backenddir = 
